projects/driver_controls_pedal/test/test_mech_brake.c

```diff
@@ -137,32 +138,32 @@ void test_mech_brake_percentage_in_pressed_then_unpressed_zone(void) {
   Event e;
 
   TEST_ASSERT_EQUAL(STATUS_CODE_OK, mech_brake_get_position(&s_mech_brake_storage, &position));
-  delay_ms(5);
+  delay_ms(10);
   // Consume the backlog of events.
   while (STATUS_CODE_OK == event_process(&e)) {
-    TEST_ASSERT_EQUAL(INPUT_EVENT_MECHANICAL_BRAKE_PRESSED, e.id);
+    TEST_ASSERT_EQUAL(INPUT_EVENT_PEDAL_MECHANICAL_BRAKE_PRESSED, e.id);
   }
 
   s_mocked_reading = 2088;  // 50 %
   TEST_ASSERT_EQUAL(STATUS_CODE_OK, mech_brake_get_position(&s_mech_brake_storage, &position));
-  delay_ms(5);
+  delay_ms(10);
```

Comment:
You can make this 10 second delay a variable so you don't need to change it everywhere next time